matrix-sdk-core/src/main/java/org/matrix/androidsdk/rest/model/WellKnown.kt

```diff
@@ -39,4 +50,35 @@ class WellKnown {
     @JvmField
     @SerializedName("m.identity_server")
     var identityServer: WellKnownBaseConfig? = null
+
+
+    @JvmField
+    @SerializedName("m.integrations")
+    var integrations: Map<String, *>? = null
+
+    /**
+     * Returns the list of integration managers proposed
+     */
+    fun getIntegrationManagers(): List<WellKnownManagerConfig> {
+        val managers = ArrayList<WellKnownManagerConfig>()
+        integrations?.get("managers")?.let {
+            (it as? ArrayList<*>)?.let { configs ->
+                configs.forEach { config ->
+                    (config as? Map<*, *>)?.let { map ->
+                        val apiUrl = map["api_url"] as? String
```

Comment:
`takeIf { it.startsWith("https://") }` can be more elegant than the if condition

Reply:
will update in following PR